examples/braket_features/Simulating_Advanced_OpenQASM_Programs_with_the_Local_Simulator.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    gate majority a, b, c {
</pre><p>Should we actually invoke this gate too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Yeah I'll make that change</p>